x/protocolpool/keeper/keeper.go

```diff
@@ -322,11 +331,11 @@ func (k Keeper) claimFunds(ctx context.Context, recipient sdk.AccAddress) (amoun
 	return amount, nil
 }
 
-func (k Keeper) getClaimableFunds(ctx context.Context, recipient sdk.AccAddress) (amount sdk.Coin, err error) {
+func (k Keeper) getClaimableFunds(ctx context.Context, recipient sdk.AccAddress, recipientAddr string) (amount sdk.Coin, err error) {
```

Comment:
ditto

Reply:
This is called from claimFunds, which just converted the string address to sdk.AccAddress. If both are not passed, we just have to convert it again.
I'm not against it, but I don't know how performance can be affected.

Reply:
As it doesn't involve complex operations and inconsistencies, I think we could just call it twice to maintain consistency with other functions